Lib/Portability.hpp

```diff
@@ -19,4 +19,12 @@ static_assert(
     "Vampire assumes that there are 8 bits in a `char`"
 );
 
+#define VAMPIRE_PERF_EXISTS 0
```

Comment:
Move this preprocessor here so you don't have to include `Timer.hpp` just for this.